pkgs/os-specific/linux/librist/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub, meson, ninja, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "librist";
+  version = "0.2.6";
+
+  src = fetchFromGitHub {
+    owner = "nanake";
+    repo = "librist";
+    rev = "v${version}";
+    sha256 = "sha256-EgWEMRiBJGq7ILmRqdL7Ok6/FhZNmYgiYyWHUhTZqCA=";
+  };
+
+  mesonFlags = [
+    "--default-library=static"
+  ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+
+  meta = with lib; {
+    description = "A library that can be used to easily add the RIST protocol to your application.";
+    homepage = "https://code.videolan.org/rist/librist";
+    license = with licenses; [ lgpl21Plus gpl2Plus ];
+    maintainers = with maintainers; [ sebtm ];
+    platforms = platforms.linux;
```

platforms.linux is too restrictive, I think `platforms.all` is fine until there's evidence to the contrary.

Also this should be in the `pkgs/development/libraries` directory, as this is not a linux specific library: https://nixos.org/manual/nixpkgs/stable/#sec-hierarchy

```suggestion
    platforms = platforms.all;
```